www/src/AdminBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -308,6 +308,34 @@ public function testFormsaveActionWithoutAdminPermission()
     }
 
     /**
+     * tests the formsaveAction with missing id parameter
+     */
+    public function testFormsaveActionWithMissingIdParameter()
+    {
+        $request = $this->getMockBuilder("Symfony\Component\HttpFoundation\Request")->disableOriginalConstructor()->getMock();
```

Comment:
Nutze hier bitte Request::class und ParameterBag::class. Das hat zum einen den Vorteil, dass der Code übersichtlicher wird, man dadurch eine direkte Verbindung in PhpStorm hat und das Refactoring ermöglicht wird.